ex2/ex2.c

```diff
@@ -8,7 +8,17 @@
 
 int main(void)
 {
-    // Your code here 
+    FILE *fp;
+    fp = fopen("./text.txt", "w");
+     int spoon = fork();
+     if (spoon < 0) { 
+        fprintf(stderr, "fork failed\n");
+        exit(1);
+    } else if (spoon == 0) {
+        fputs("Write Child\n", fp);
```

Comment:
what is the difference between `fputs` and `fprintf`?

Reply:
From my understanding they both do pretty much the same thing and print out a string to the stream